frontend/src/app/shared/form/manifest-builder/manifest-builder.component.html

```diff
@@ -0,0 +1,530 @@
+<div class="manifest-builder">
+  <form [formGroup]="form">
+    <div class="row">
+      <div class="col-md-6">
+        <div class="form-group">
+          <label class="form-label" for="manifest-key" translate="configuration.oibus.manifest.transformers.manifest-builder.key"></label>
+          <input type="text" class="form-control" formControlName="key" id="manifest-key" />
+        </div>
+      </div>
+      <div class="col-md-6">
+        <div class="form-group">
+          <label
+            class="form-label"
+            for="manifest-translation-key"
+            translate="configuration.oibus.manifest.transformers.manifest-builder.translation-key"
+          ></label>
+          <input type="text" class="form-control" formControlName="translationKey" id="manifest-translation-key" />
+        </div>
+      </div>
+    </div>
```

Comment:
Let's remove this part and automatically wrap the manifest inside an option when retrieving the form value